test/braille_test.rb

```diff
@@ -0,0 +1,32 @@
+require "minitest/autorun"
+require "minitest/pride"
+require "mocha/minitest"
+require "./lib/braille"
+require "./lib/night_writer"
+require "./lib/invisible_ink"
+
+class BrailleTest < Minitest::Test
+  def setup
+    @braille = Braille.new
+
+  end
+  def test_it_exists
+    assert_instance_of Braille,
+    @braille
+  end
+
+  def test_it_has_attributes
+    @braille.stubs(:dictionary).returns({})
```

Comment:
I attempted to have a test that shows this is an empty hash at the beginning and then gets more information once the set_braille_dictionary is called.